python/lsst/resources/_resourcePath.py

```diff
@@ -1299,7 +1299,7 @@ def open(
 
         Returns
         -------
-        cm : `contextlib.ContextManager`
+        cm : `contextlib.contextmanager`
```

Comment:
This is a function, not a type.  I think you want `contextlib.AbstractContextManager`

Reply:
or maybe change it to Yields an iterator of type resource handle protocol.

Reply:
I don't think so; the use of a decorator that changes the return type should be an implementation detail as far as the docs are returned.  MyPy is smart enough to transform the return type via the decorator but Sphinx cannot.

Reply:
Ok. Will change it back. In much of our other documentation I think we say what the context manager yields and don't declare the decorated thing.